modules/smtp/functions.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+
+if (!defined('DEBUG_MODE')) { die(); }
+
+if (!hm_exists('add_and_connect_to_smtp_server')) {
+    function connect_to_smtp_server($address, $name, $port, $user, $pass, $tls, $context) {
+        Hm_SMTP_List::init($context->user_config, $context->session);
```

Comment:
Hm_Handler_load_smtp_servers_from_config is used to initialize the smtp list from the repo.